Lib/_collections_abc.py

```diff
@@ -811,7 +811,7 @@ def __getitem__(self, key):
         raise KeyError
 
     def get(self, key, default=None):
-        'D.get(k[,d]) -> D[k] if k in D, else d.  d defaults to None.'
+        'D.get(k[,d]) -> D[k] if k in D else d.  d defaults to None.'
```

Comment:
I may be wrong but isn't this a comment? hence looks like punctuation, but am no native English speaker.

Reply:
Hmm. Didn't think about that. Python looks too similar to English. Do you think we should close the PR then?

Reply:
I have rolled back this change. And updated this PR to make docstrings' quotes consistent within the module (which I was going to do in another PR).